vue/sbc-common-components/src/components/NotificationPanel.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <div
+    v-if="showNotifications"
+    v-on:clickout="emitClose()"
+  >
+    <v-overlay></v-overlay>
+    <v-navigation-drawer
+      right
+      app
+      :width="440"
+      style="top: 70px; max-height: calc(100% - 0);"
+    >
+      <v-app-bar
+        flat
+        outlined
+        color="#FCBA19"
```

Comment:
It's better to not have hex colour values in components.

You can use a theme colour name here, since one was declared here:
https://github.com/bcgov/sbc-common-components/blob/master/vue/sbc-common-components/src/plugins/vuetify.ts